docs/documentation/head/setup.md

```diff
@@ -42,3 +42,6 @@ Where MM is the major version, nn is the minor version and pp is the patch versi
 Versions for JDBC3 and lower can be found [here](https://jdbc.postgresql.org/download.html#others)
 This is a very brief outline of how to build the driver. Much more detailed information can be 
 found on the [github repo](https://github.com/pgjdbc/pgjdbc/blob/master/CONTRIBUTING.md)
+
+Even though the JDBC driver should be build with Gradle, for situations, where use of Gradle is not possible, e.g., when building pgjdbc for distributions, the pgjdbc provides a convenience source release artifact **-src.tar.gz**- a Maven based project. The Maven based project contains a minimal version of the JDBC driver with reduced feature and test set and is still validly buildable within the Maven build environment.
+Minimal version of the JDBC driver is available on ["Maven central repository"](https://repo1.maven.org/maven2/org/postgresql/postgresql/) or can be created with **gradlew -d :postgresql:sourceDistribution -Prelease**
```

Comment:
This is a bit confusing. I think you mean that the source version is available here ... ?

Reply:
Thanks, corrected in the second commit.